pkg/sql/logictest/testdata/logic_test/schema_repair

```diff
@@ -0,0 +1,61 @@
+subtest lost-table-data
+
+statement ok 
+CREATE TABLE lostdata (v INT8)
+
+statement ok
+INSERT INTO lostdata VALUES (3);
+
+statement ok
+INSERT INTO lostdata VALUES (5);
+
+statement ok
+INSERT INTO lostdata VALUES (23);
+
+let $t_id
+SELECT id FROM system.namespace WHERE name = 'lostdata';
+
+query I
+SELECT * FROM crdb_internal.lost_descriptors_with_data;
+----
+
+let $json
+WITH
+	descs
+		AS (
+			SELECT
+				id,
+				crdb_internal.pb_to_json(
+					'cockroach.sql.sqlbase.Descriptor',
+					descriptor
+				)
+					AS descriptor
+			FROM
+				system.descriptor
+		)
+SELECT
+	descriptor
+FROM
+	descs
+WHERE
+	id = $t_id;
```

Comment:
Can you reformat this through https://sqlfum.pt/ please?

Reply:
That actually went through sqlfum.pt already. Passed it again and see no difference.

Reply:
You're right, sorry about that. I think I messed with the defaults in sqlfumpt. Whoops.